config/core/test-configs.yaml

```diff
@@ -425,6 +434,10 @@ test_plans:
     params:
       job_timeout: '10'
       kselftest_collections: "vm"
+    filters:
+      - passlist:
+        - defconfig: ['kselftest']
+        - kernel: ['v3.', 'v4.', 'v5', 'v6.0', 'v6,1', 'v6.2']
 
```

Comment:
Changes required:
```
-        - defconfig: ['kselftest']
-        - kernel: ['v3.', 'v4.', 'v5', 'v6.0', 'v6,1', 'v6.2']
+          defconfig: ['kselftest']
+          kernel: ['v3.', 'v4.', 'v5', 'v6.0', 'v6,1', 'v6.2']
```
So this fragment will look like:
```
  kselftest-vm:
    <<: *kselftest
    params:
      job_timeout: '10'
      kselftest_collections: "vm"
    filters:
      - passlist:
          defconfig: ['kselftest']
          kernel: ['v3.', 'v4.', 'v5', 'v6.0', 'v6,1', 'v6.2']
```

Reply:
Also it should be `v5.`, or we might match kernel v50 one day...

Reply:
And there's a typo in `v6.1` (`v6,1`)

Reply:
The benevolent dictator in charge of the Linux versioning scheme might decide to increase the major version at every release starting with v10 or something arbitrary, a bit like "hey we're now at v2.6.39 so let's jump to v3.0"...